x/wasm/internal/keeper/test_common.go

```diff
@@ -59,132 +113,149 @@ var TestingStakeParams = staking.Params{
 }
 
 type TestKeepers struct {
-	AccountKeeper auth.AccountKeeper
-	StakingKeeper staking.Keeper
+	AccountKeeper authkeeper.AccountKeeper
+	StakingKeeper stakingkeeper.Keeper
 	WasmKeeper    Keeper
-	DistKeeper    distribution.Keeper
-	SupplyKeeper  supply.Keeper
-	GovKeeper     gov.Keeper
-	BankKeeper    bank.Keeper
+	DistKeeper    distributionkeeper.Keeper
+	BankKeeper    bankkeeper.Keeper
+	GovKeeper     govkeeper.Keeper
 }
 
 // encoders can be nil to accept the defaults, or set it to override some of the message handlers (like default)
-func CreateTestInput(t *testing.T, isCheckTx bool, tempDir string, supportedFeatures string, encoders *MessageEncoders, queriers *QueryPlugins) (sdk.Context, TestKeepers) {
-	keyContract := sdk.NewKVStoreKey(wasmtypes.StoreKey)
-	keyAcc := sdk.NewKVStoreKey(auth.StoreKey)
-	keyStaking := sdk.NewKVStoreKey(staking.StoreKey)
-	keySupply := sdk.NewKVStoreKey(supply.StoreKey)
-	keyDistro := sdk.NewKVStoreKey(distribution.StoreKey)
-	keyParams := sdk.NewKVStoreKey(params.StoreKey)
-	tkeyParams := sdk.NewTransientStoreKey(params.TStoreKey)
+func CreateTestInput(t *testing.T, isCheckTx bool, supportedFeatures string, encoders *MessageEncoders, queriers *QueryPlugins) (sdk.Context, TestKeepers) {
+	tempDir, err := ioutil.TempDir("", "wasm")
+	require.NoError(t, err)
+	t.Cleanup(func() { os.RemoveAll(tempDir) })
```

Comment:
Oh... I never knew about `t.Cleanup()`. That is a really nice test function! 

Ignore my comment above